src/Serilog/LoggerConfiguration.cs

```diff
@@ -185,9 +179,18 @@ public Logger CreateLogger()
                 overrideMap = new LevelOverrideMap(_overrides, _minimumLevel, _levelSwitch);
             }
 
+            var disposableSinks = _logEventSinks.Concat(_auditSinks).OfType<IDisposable>().ToList();
```

Comment:
ToArray might be microscopically better?

Reply:
There is no reason to not making it microscopically better :) Done.